### PR TITLE
Env storage

### PR DESCRIPTION
**Pull request by seromenho**
Adds per environment user data.
This means developers can now switch between environments and have a different login on each one.

### CODE REVIEW

File: fliplet-organization.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 const auth = require('./lib/auth');
 const organizations = require('./lib/organizations');
-const configstore = require('./lib/configstore');
 
 const organizationId = Number(process.argv[2]);
+
 if (!organizationId) {
   console.log('Organization reset');
-  return configstore.delete('organization');
+  return config.set('organization', null);
```
Comment (squallstar) [inline fliplet-organization.js]: where is config defined?

File: lib/config.js [hunk 2]
```diff
@@ -12,5 +12,10 @@ var correctEnvironment = getCorrectEnvironment(envStored);
 config = require(`../config/${correctEnvironment}.json`);
```
Comment (squallstar) [inline lib/config.js]: replace "correct" with "selected" or "current"

File: fliplet-organization.js (hunk 1 above)
Comment (seromenho) [inline fliplet-organization.js]: True. Why it doesn't trigger an error. wtf

File: lib/config.js (hunk 2 above)
Comment (seromenho) [inline lib/config.js]: ty

**Comment by seromenho** [comment]
Merged and published to npm

**Pull request merged**